deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/Clusters.java

```diff
@@ -124,6 +133,38 @@ public void deleteCluster(@Context SecurityContext sc,
     }
 
     @PUT
+    @Path("/actions")
+    public void replaceCluster(@Context SecurityContext sc,
+                               @PathParam("envName") String envName,
+                               @PathParam("stageName") String stageName,
+                               @QueryParam("type") String type) throws Exception {
+        EnvironBean envBean = Utils.getEnvStage(environDAO, envName, stageName);
+        authorizer.authorize(sc, new Resource(envBean.getEnv_name(), Resource.Type.ENV), Role.OPERATOR);
+        String operator = sc.getUserPrincipal().getName();
+        ActionType actionType = ActionType.valueOf(ActionType.class, type.toUpperCase());
+        switch (actionType) {
+            case ENABLE:
+                clusterHandler.replaceCluster(envName, stageName);
+                break;
+            case PAUSE:
+                clusterHandler.pause(envName, stageName);
+                break;
+            case RESUME:
+                clusterHandler.resume(envName, stageName);
+                break;
+            case CANCEL:
+                clusterHandler.complete(envName, stageName);
```

Comment:
suggest to use the following names:
replace/pauseReplace/resumeReplace/cancelReplace